### ISSUE TITLE
Install without npmjs ?

### ISSUE THREAD

**Issue by phis5533** [DaGolle/homebridge-rfxtrx#1]
Hello @DaGolle ,

your project sounds really good as far as I'm a Homebridge and RFXtrx user !!
But for the moment, I use only npmjs to install package in my Homebridge configuration and your code is not available on npmjs.com....
Can you detail how I have to install your package in my configuration ?
/usr/local/lib/node_modules/??
Thank you
KR.
Philippe

**Comment by DaGolle** [comment]
Hi @phis5533,

Just published homebridge-rfxtrx to npmjs, let me know if it works for you.
FYI, in the meantime I have switched to Home Assistant, so I'm no longer using/developing it myself.

Cheers,
DaGolle

**Comment by phis5533** [comment]
Hi DaGolle,

Thx very much for your work…

For the moment I’m trying to setup a RFY connection (Somfy) Window Covering , but doesn’t work…
I’ll try to debug and let you know ;-)
Good luck for your others work !

KR

PhV

Le 4 nov. 2018 à 17:19, David VD <notifications@github.com> a écrit :

Hi @phis5533 <https://github.com/phis5533>,

Just published homebridge-rfxtrx to npmjs, let me know if it works for you.
FYI, in the meantime I have switched to Home Assistant, so I'm no longer using/developing it myself.

Cheers,
DaGolle

**Comment by phis5533** [comment]
Hi David,

I've succeeded in running your plugin with Homebridge, but I'm not sure that Somfy RTS can work with it because it doesn't communicate both way.

Strange thing, I can't make it work whiteout outlets parameters... weird....:-) 

Anyway, thank you again for your work.
KR.
Philippe

**Comment by DaGolle** [comment]
Hi Philippe,

I don't have any Somfy RTS compatible devices, so I wouldn't know how they work. The devices I own just need to receive a command, which I can send with the RFXTRX.
The pairing process is usually a manual process where the RFXTRX sends the command and the device is in "learning" mode. I used the Windows app from RFXCOM for that and then copied the codes I used to the config.json file.
Not sure if that process is the same for Somfy though.

The design of what I wrote is very simple and I just provided 2 "HomeBridge Accessories" for the rfxcom library: WindowCovering and Outlet.
Most of the programming I did was around the logic for the WindowCovering, which uses a timer to determine wether it's open or closed in HomeKit (since there is no feedback from the device itself about it's state).
Make sure you check out the documentation for rfxcom (https://github.com/rfxcom/node-rfxcom) as well.

Good luck!
